### PR TITLE
Dark Mode - NxColorPicker - RSC-1517

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-1517

### CODE REVIEW

File: lib/src/components/NxColorPicker/NxColorPicker.scss [hunk 1]
```diff
@@ -21,13 +21,13 @@
 
     &:focus-within {
       .nx-color-picker__outline {
-        stroke: var(--nx-swatch-teal-35);
+        stroke: var(--nx-color-color-picker-border-focus);
```
Comment (EmilyBermudez88) [inline lib/src/components/NxColorPicker/NxColorPicker.scss:24]:
```suggestion
        stroke: var(--nx-color-picker-border-focus);
```

File: gallery/src/styles/CssVariables/CssVariablesPage.tsx [hunk 2]
```diff
@@ -952,7 +952,13 @@ const CssVariablesPage = () => {
               The value used for line colors within the <NxCode>NIVO_THEME</NxCode> export.
             </ColorDocRow>
             <ColorDocRow colorVar="--nx-color-chart-text">
-              The value used for text colors within the <NxCode>NIVO_THEME</NxCode> export.
+              The value used for text colors within the <NxCode>NIVO_THEME</NxCode>.
```
[the remark this answers is not in the document]
Comment (avnvbhatta) [reply gallery/src/styles/CssVariables/CssVariablesPage.tsx]: > was it accidental to remove 'export' from the end of the sentence?

Ah yes, changed here: 340611ad75dd42f80c18f6ea2921eda63459598d

**Review by EmilyBermudez88 — APPROVED** [review]
LGTM

File: lib/src/base-styles/_nx-colors.scss [hunk 3]
```diff
@@ -293,6 +293,10 @@
   --nx-color-status-indicator-error-icon-background: var(--nx-swatch-red-65);
   --nx-color-status-indicator-error-icon-border: var(--nx-color-alert-error);
 
+  // NxColorPicker
+  --nx-color-color-picker-border-focus: var(--nx-swatch-teal-35);
```
Comment (rpokorny) [inline lib/src/base-styles/_nx-colors.scss:297]: A seeing a discrepancy in the mocks: at least one spot has the focus ring as blue-40, though most have it as teal-35

File: lib/src/components/NxColorPicker/NxColorPicker.scss [hunk 4]
```diff
@@ -21,13 +21,13 @@
...
       }
     }
 
     &:hover {
       .nx-color-picker__color {
-        stroke: var(--nx-color-interactive-border-hover);
+        stroke: var(--nx-color-text-stark);
```
Comment (rpokorny) [inline lib/src/components/NxColorPicker/NxColorPicker.scss:30]: It seems coincidental that this results in the correct values, let's make another variable

**Pull request merged**